flink/json-jobmanager.yml

```diff
@@ -0,0 +1,224 @@
+
+  - name: jobmanager_nonheap_committed
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.NonHeap.Committed)]
+    metric:
+      - name: jobmanager.nonheap.committed
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: committed non-heap
+        description: The amount of non-heap memory guaranteed to be available to the JVM
+
+  - name: jobmanager_direct_max
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.TotalCapacity)]
+    metric:
+      - name: jobmanager.direct.max
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: direct used
+        description: The total capacity of all buffers in the direct buffer pool
+
+  - name: jobmanager_direct_used
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.MemoryUsed)]
+    metric:
+      - name: jobmanager.direct.used
+        source: $.[?(@.id=Status.JVM.Memory.Direct.MemoryUsed)].value
+        type: long_gauge
+        unit: bytes
+        label: direct used
+        description: The amount of memory used by the JVM for the direct buffer pool
+
+  - name: jobmanager_direct_count
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.Count)]
+    metric:
+      - name: jobmanager.direct.count
+        source: value
+        type: long_gauge
+        label: direct count
+        description: The number of buffers in the direct buffer pool
+
+  - name: jobmanager_mapped_max
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.TotalCapacity)]
+    metric:
+      - name: jobmanager.mapped.max
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: mapped count
+        description: The number of buffers in the mapped buffer pool
+
+  - name: jobmanager_mapped_used
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.MemoryUsed)]
+    metric:
+      - name: jobmanager.mapped.used
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: mapped used
+        description: The amount of memory used by the JVM for the mapped buffer pool
+
+  - name: jobmanager_mapped_count
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.Count)]
+    metric:
+      - name: jobmanager.mapped.count
+        source: value
+        type: long_gauge
+        label: mapped count
+        description: The number of buffers in the mapped buffer pool
+
+  - name: jobmanager_threads_count
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Threads.Count)]
+    metric:
+      - name: jobmanager.threads.count
+        source: value
+        type: long_gauge
+        label: The total number of live threads
+
+  - name: jobmanager_cpu_time
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.CPU.Time)]
+    metric:
+      - name: jobmanager.cpu.time
+        source: value
+        type: long_gauge
+        label: cpu time
```

Comment:
This is cpu time in... ms? Adjust the name, label, description + add a unit.

Reply:
Add unit "ms", changed type to counter, docs describe metric type is gauge, but i see it's value increase
not sure how to adjust name, label?
I got "description" from 
https://ci.apache.org/projects/flink/flink-docs-release-1.7/monitoring/metrics.html

Reply:
Hm, maybe again error in their docs. Adding a unit should be enough.

Reply:
Added unit, description